sdk/appconfiguration/test-resources.json

```diff
@@ -1,39 +1,46 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
-    "contentVersion": "1.0.0.0",
-    "parameters": {
-        "baseName": {
-            "type": "string",
-            "defaultValue": "[resourceGroup().name]",
-            "metadata": {
-                "description": "The base resource name."
-            }
-        }
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "baseName": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().name]",
+      "metadata": {
+        "description": "The base resource name."
+      }
     },
-    "variables": {
-        "configurationApiVersion": "2019-02-01-preview",
-        "accountName": "[parameters('baseName')]",
-        "endpoint": "[concat('https://', parameters('baseName'), '.azconfig.io')]"
+    "sku": {
+      "defaultValue": "Standard",
+      "type": "string"
+    }
+  },
+  "variables": {
+    "configurationApiVersion": "2019-02-01-preview",
```

Comment:
Should the version be updated to `2019-10-01` now that it's real? 

Reference: https://docs.microsoft.com/en-us/azure/azure-app-configuration/quickstart-resource-manager#deploy-an-azure-app-configuration-store